assets/js/modules/subscribe-with-google/bulk-edit.js

```diff
@@ -0,0 +1,24 @@
+// eslint-disable-next-line no-alert
+alert( 'Hello world' );
```

Comment:
See my other comment, please move this file into `assets/js` like the other entry points. Not sure why this doesn't work right now, but maybe try adding some imports, potentially wrap this in a `domReady` for example.

The `googlesitekit-idea-hub-post-list.js` file is loaded in the same context, so that should be working.